src/hocs/withCheckNetworkConnection/withCheckNetworkConnection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Alert } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { selectors } from 'app/state/electrumX';
+
+type CheckNetworkConnectionArgs = (...args: unknown[]) => void;
+
```

Comment:
I just noticed that the name is misleading - it's not about args, but it's just a callback definition. So, maybe it'd be better to rename it to `CheckNetworkConnectionCallback`? Also, consider exporting it.

Reply:
done